src/main/java/org/folio/inventory/domain/instances/Dates.java

```diff
@@ -30,13 +32,26 @@ public static JsonObject datesToJson(Dates dates) {
     return json;
   }
 
-  public static Dates datesFromJson(JsonObject datesJson) {
-    if (datesJson == null) {
+  public static JsonObject retrieveDatesFromJson(JsonObject targetInstance) {
+    JsonObject dates;
+    dates = targetInstance.getJsonObject(DATES_KEY);
```

Comment:
I think it can be done in one line

Reply:
@Aliaksandr-Fedasiuk  Done. Could you please double-check it?